php/commands/site.php

```diff
@@ -1403,6 +1403,82 @@ public function service_level($args, $assoc_args) {
   }
 
   /**
+   * Manage site organization tags
+   *
+   * ## OPTIONS
+   *
+   * <add|remove>
+   * : subfunction to run
+   *
+   * [--site=<site>]
+   * : Site's name
+   *
+   * [--org=<name|id>]
+   * : Organization to apply tag with
+   *
+   * [--tag=<tag>]
+   * : Tag to add or remove
+   *
+   * @subcommand tag
```

Comment:
I think the spec says "tag" but I wonder if it should be "tags" (considering we just renamed "backup" to the plural). Feel free to ship it as is and revisit later. I'm happy with the functionality!